ui/app/AppLayouts/Profile/popups/BackupSeedModal.qml

```diff
@@ -96,12 +96,20 @@ StatusStackModal {
         }
     }
 
+    // split subHeaderPadding into clip and non-clip parts
+    subHeaderPadding: 8
+    readonly property int nonClipSubHeaderPadding: 8
```

Comment:
If it's private variable is better to put it into QtObject d

Reply:
I decided to put it here for readability (keeping it where it's used). Imo it's not harmful because it's read-only.